components/uid.rst

```diff
@@ -158,8 +158,40 @@ following methods to create a ``Uuid`` object from it::
     $uuid = Uuid::fromBase58('TuetYWNHhmuSQ3xPoVLv9M');
     $uuid = Uuid::fromRfc4122('d9e7a184-5d5b-11ea-a62a-3499710062d0');
 
-You can also use the ``UuidFactory`` to generate UUIDs. First, you may
-configure the behavior of the factory using configuration files::
+You can also use the ``UuidFactory`` to generate UUIDs. Inject the factory in
+your services and use it as follows:
+
+    namespace App\Service;
+
+    use Symfony\Component\Uid\Factory\UuidFactory;
+
+    class FooService
+    {
+        public function __construct(
+            private UuidFactory $uuidFactory,
+        ) {
+        }
+
+        public function generate(): void
+        {
+            $uuid = $this->uuidFactory->create();
+
+            $nameBasedUuid = $this->uuidFactory->nameBased(/** ... */);
```

Comment:
The `nameBased()`, `timeBased()` and `randomBased()` methods don't return a UUID directly, but return a factory instead on which we then have to call `create()`. So this example would have to look like this:


```suggestion
            $nameBasedUuid = $this->uuidFactory->nameBased(/** ... */)->create();
```